INSTALL.md

```diff
@@ -28,19 +10,19 @@ platforms:
 * Mac OS X 10.11.4 El Capitan + Xcode 7.3 + OS X 10.11 SDK + Skia (without GPU)
 * Linux + gcc 4.8 with some C++11 support
 
-# Get the source code
+To compile LibreSprite you will need:
 
-You can get the source code by downloading a zip or tar.gz snapshot of the
-master branch of the LibreSprite repository:
+* [CMake](http://www.cmake.org/) (3.4 or greater)
+* [Ninja](https://ninja-build.org)
+* [vcpkg](https://vcpkg.io/en/getting-started.html) (Windows and Mac OS only)
```

Comment:
(nitpicking)
if without "X", it's "MacOS" without spaces

Reply:
Should we change all instances of Mac OS X to MacOS, or vice versa?

Reply:
I've changed it to MacOS.